src/app/components/FacetSidebar/FacetSidebar.jsx

```diff
@@ -179,11 +179,13 @@ class FacetSidebar extends React.Component {
                 return (
                   <label
                     key={j}
+                    id={`${facet.field}-${valueLabel}`}
                     htmlFor={`${facet.field}-${valueLabel}`}
                     className={`nypl-bar_${percentage}`}
                   >
                     <input
                       id={`${facet.field}-${valueLabel}`}
+                      aria-labelledby={`${facet.field} ${valueLabel}`}
```

Comment:
i think i can do something with this but i really need the number to come _before_ the text... that's what allows me to float it the way it is

so, if React will oblige, i'd wish something like:
````html
<label for="subject-fiction" class="nypl-bar_22">
  <input id="subject-fiction" type="checkbox" name="subject" value="fiction">
  <span class="nypl-facet-count">75</span>
  <span>fiction</span>
</label>
````